src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -144,4 +147,25 @@ public String getPagedResponseTypeInnerName(
   public String getFullyQualifiedApiWrapperClassName(Interface service) {
     return getPackageName() + "." + getApiWrapperClassName(service);
   }
+
+  @Override
+  public String getRandomStringValue(String randomString) {
+    String randomValueGen = "System.currentTimeMillis()";
+    String delimiter = ",";
+    String[] splitted =
+        FieldStructureParser.stripQuotes(randomString)
```

Comment:
This is general inject method in `JavaSurfaceNamer` so it would be risky to make such an assumption on the input that it is not quoted.  I would prefer to keep the quote stripping here for robustness.